src/locale/en.json

```diff
@@ -94,5 +94,7 @@
   "bug_type_message":"Please enter the type of request",
   "bug_type_bug":"Bug",
   "bug_type_feature":"Feature Request",
-  "funding_paragraph": "The Databus project is developed by the Hasadna for public knowledge, by the contributions of volunteers, and based upon"
+  "funding_paragraph": "The Databus project is developed by the Hasadna for public knowledge, by the contributions of volunteers, and based upon",
```

Comment:
Space removed by purpose?

Reply:
> Space removed by purpose?

No. Maybe Lint complained, but I can't even see what space we are talking about... just that there is a change in "funding_paragraph" that I did not mean to make.